Grape vine

HamishBrownPFR opened the pull request:
working on #1292  @junqi108 I have fixed the problem with your pull request.  Will explain how later

junqi108 commented:
OK, thanks very much, Hamish. It is getting difficult to merge now.
Cheers!

On 8 Nov 2016 17:13, "Hamish Brown" <notifications@github.com> wrote:

working on #1292 <https://github.com/APSIMInitiative/ApsimX/issues/1292>
@junqi108 <https://github.com/junqi108> I have fixed the problem with your
pull request. Will explain how later
------------------------------
You can view, comment on, or merge this pull request online at:

  https://github.com/APSIMInitiative/ApsimX/pull/1350
Commit Summary

   - added the berry model
   - Merge branch 'master' of https://github.com/APSIMInitiative/ApsimX
   into GrapeVine
   - Change biomass removal folder into class in berry organ, Give Biomass
   removal class valid parent tag so it can be added to organs

File Changes

   - *M* Models/Core/Model.cs
   <https://github.com/APSIMInitiative/ApsimX/pull/1350/files#diff-0> (1)
   - *M* Models/Models.csproj
   <https://github.com/APSIMInitiative/ApsimX/pull/1350/files#diff-1> (1)
   - *A* Models/Plant/Functions/DemandFunctions/BerryFillingRateFunction.cs
   <https://github.com/APSIMInitiative/ApsimX/pull/1350/files#diff-2> (100)
   - *M* Models/Plant/Library/BiomassRemoval.cs
   <https://github.com/APSIMInitiative/ApsimX/pull/1350/files#diff-3> (2)
   - *M* Prototypes/Grapevine/grapevine.apsimx
   <https://github.com/APSIMInitiative/ApsimX/pull/1350/files#diff-4>
   (1208)

Patch Links:

   - https://github.com/APSIMInitiative/ApsimX/pull/1350.patch
   - https://github.com/APSIMInitiative/ApsimX/pull/1350.diff

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/APSIMInitiative/ApsimX/pull/1350>, or mute the thread
<https://github.com/notifications/unsubscribe-auth/AMnCoo05q75DvAftcI6-8blUKNtOVLE9ks5q7_bRgaJpZM4KsBDU>
.

HamishBrownPFR commented:
No not difficult.  Just something was change in parallel that we hadn’t picked up on.  Easy enough to fix

From: Junqi ZHU [mailto:notifications@github.com]
Sent: Tuesday, 8 November 2016 5:15 p.m.
To: APSIMInitiative/ApsimX
Cc: Hamish Brown; Author
Subject: Re: [APSIMInitiative/ApsimX] Grape vine (#1350)

OK, thanks very much, Hamish. It is getting difficult to merge now.
Cheers!

On 8 Nov 2016 17:13, "Hamish Brown" <notifications@github.com<mailto:notifications@github.com>> wrote:

working on #1292 <https://github.com/APSIMInitiative/ApsimX/issues/1292>
@junqi108 <https://github.com/junqi108> I have fixed the problem with your
pull request. Will explain how later
------------------------------
You can view, comment on, or merge this pull request online at:

https://github.com/APSIMInitiative/ApsimX/pull/1350
Commit Summary

- added the berry model
- Merge branch 'master' of https://github.com/APSIMInitiative/ApsimX
into GrapeVine
- Change biomass removal folder into class in berry organ, Give Biomass
removal class valid parent tag so it can be added to organs

File Changes

- *M* Models/Core/Model.cs
<https://github.com/APSIMInitiative/ApsimX/pull/1350/files#diff-0> (1)
- *M* Models/Models.csproj
<https://github.com/APSIMInitiative/ApsimX/pull/1350/files#diff-1> (1)
- *A* Models/Plant/Functions/DemandFunctions/BerryFillingRateFunction.cs
<https://github.com/APSIMInitiative/ApsimX/pull/1350/files#diff-2> (100)
- *M* Models/Plant/Library/BiomassRemoval.cs
<https://github.com/APSIMInitiative/ApsimX/pull/1350/files#diff-3> (2)
- *M* Prototypes/Grapevine/grapevine.apsimx
<https://github.com/APSIMInitiative/ApsimX/pull/1350/files#diff-4>
(1208)

Patch Links:

- https://github.com/APSIMInitiative/ApsimX/pull/1350.patch
- https://github.com/APSIMInitiative/ApsimX/pull/1350.diff

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/APSIMInitiative/ApsimX/pull/1350>, or mute the thread
<https://github.com/notifications/unsubscribe-auth/AMnCoo05q75DvAftcI6-8blUKNtOVLE9ks5q7_bRgaJpZM4KsBDU>
.


—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/APSIMInitiative/ApsimX/pull/1350#issuecomment-259041336>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AG9Tcq58vR-q18vuUcUQp8xu6n-TylCuks5q7_dCgaJpZM4KsBDU>.

The contents of this e-mail are confidential and may be subject to legal privilege.
 If you are not the intended recipient you must not use, disseminate, distribute or
 reproduce all or any part of this e-mail or attachments.  If you have received this
 e-mail in error, please notify the sender and delete all material pertaining to this
 e-mail.  Any opinion or views expressed in this e-mail are those of the individual
 sender and may not represent those of The New Zealand Institute for Plant and
 Food Research Limited.
